### PR TITLE
Amend deploy-flux.sh

### PR DESCRIPTION
**Pull request by zc-hmcts**
- add toffee to workaround

### CODE REVIEW

File: bootstrap/deploy-flux.sh [hunk 1]
```diff
@@ -57,6 +57,7 @@ EOF
         kubectl -n flux-system wait --for condition=established --timeout=60s "customresourcedefinition.apiextensions.k8s.io/$crd"
     done
     
+    kubectl apply -f https://raw.githubusercontent.com/hmcts/sds-flux-config/master/apps/toffee/identity/toffee-azure-identity.yaml
```
Comment (timja) [inline bootstrap/deploy-flux.sh:60]: why?

Comment (zc-hmcts) [reply bootstrap/deploy-flux.sh:60]: unable to recognize "STDIN": no matches for kind "AzureIdentity" in version "aadpodidentity.k8s.io/v1"

Comment (timja) [reply bootstrap/deploy-flux.sh:60]: try restart the pod, or check the CRDs are ready

Comment (zc-hmcts) [reply bootstrap/deploy-flux.sh:60]: it's from the build logs, building from scratch

**Review by timja — CHANGES_REQUESTED** [review]
we can't do this

**Pull request closed without merging**